Source/Thunder/PluginHost.cpp

```diff
@@ -712,15 +712,13 @@ POP_WARNING()
                                 printf("Link: %s\n", index.Current().Remote.Value().c_str());
                                 printf("------------------------------------------------------------\n");
 
-                                RPC::Administrator::Proxies proxies;
-
-                                RPC::Administrator::Instance().Allocations(index.Current().ID.Value(), proxies);
-
-                                for (const ProxyStub::UnknownProxy* proxy : proxies) {
+                                RPC::Administrator::Instance().Allocations(index.Current().ID.Value(), [](const std::vector<ProxyStub::UnknownProxy*>& proxies) {
+                                    for (const auto& proxy: proxies) {
```

Comment:
Is this to test and see if I pay attention ;-) Please remove the printfs, and fill the proxies, cause that is a bit essential  ;-)

Reply:
Hi Pierre, the printfs are from existinng code:
https://github.com/rdkcentral/Thunder/blob/master/Source/Thunder/PluginHost.cpp#L721

Reply:
and I am not sure what you mean by filling the proxies:
the old code includes this:
RPC::Administrator::Proxies proxies; 

but i removed it as this part of the function only displays data, so I just directly use: 
[](const std::vector<ProxyStub::UnknownProxy*>& proxies)

I dont think there is a need to pass something like a new vector like
[&proxies]?
Or am i missing something?